app/javascript/packs/Notification/notification.jsx

```diff
@@ -149,6 +162,20 @@ export default class Notification extends React.Component {
 
         return (
             <>
+                {this.props.admin
+                        ? <input className="form-check-input"
+                                 type="checkbox" name="notificationsToArchive"
+                                 style={{ width: '25px',
+                                     height:'25px',
+                                     borderRadius: '5px',
+                                     border:'2px',
+                                     marginTop: '25px'}}
+                                 value={this.props.NotificationID}
+                                 checked={this.state.checked}
+                                 onChange={e => {this.handleCheckbox(e.target.value)}}/>
+
+                    : '' }
+
```

Comment:
Create new component for checkbox, becouse whole notification component is rerendered when state "checked" is updated

Reply:
Done